wodles/azure/tests/test_orm.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute
+# it and/or modify it under the terms of GPLv2
+
+
+import hashlib
+import json
+import os
+import pytest
+import sys
+
+from sqlalchemy import create_engine
+from sqlalchemy import inspect
+from sqlalchemy.exc import IntegrityError
+from unittest.mock import patch
+
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))  # noqa: E501
+with patch('sqlalchemy.create_engine', return_value=create_engine("sqlite://")):
+    import orm
+
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_last_dates_path = os.path.join(test_data_path, 'last_date_files')
+
+EXPECTED_TABLE_NAMES = ['graph', 'log_analytics', 'storage']
+
+
+@pytest.fixture(scope='function')
+def create_and_teardown_db():
+    orm.create_db()
+    yield
+    orm.Base.metadata.drop_all(orm.engine)
+
+
+@pytest.fixture(scope='function')
+def teardown_db():
+    yield
+    orm.Base.metadata.drop_all(orm.engine)
+
+
+def test_create_db(teardown_db):
+    # Check there is no tables available
+    inspector = inspect(orm.engine)
+    assert inspector.get_table_names() == []
+
+    # Create the tables
+    orm.create_db()
+
+    # Validate the tables and their structure
+    inspector = inspect(orm.engine)
+    table_names = inspector.get_table_names()
+    assert table_names == EXPECTED_TABLE_NAMES
+    for table in table_names:
+        assert [x['name'] for x in inspector.get_columns(table)] == ['md5', 'min_processed_date', 'max_processed_date']
+
+
+def test_add_get_row(create_and_teardown_db):
+    """Test the ORM is able to insert row into the different tables and retrieve them as a whole and individually as
+    expected."""
```

Comment:
That's not a parameter but a fixture. We currently do not add fixtures to our docstrings because fixtures are functions that are triggered before and after the test. Only test parameters will be present.

Reply:
I've seen it otherwise, but since the fixture is not being used as a parameter and the use is clear I think it's OK.